app/lib/utils/formatTimeString.js

```diff
@@ -0,0 +1,15 @@
+const moment = require('moment');
+
+function formatTimeString(timeString) {
+  if (timeString === '23:59' || timeString === '00:00') {
+    return 'midnight';
+  }
+  const time = moment();
```

Comment:
This block could be condensed a little and rewritten something like i.e. the moment can be created with the time initially and I think the comparison of `time.minutes() === 0` could just be `minutes === 0`.
```js
const [hours, minutes] = timeString.split(':').map(Number);
const time = moment('${hours}:${minutes}', 'HH:mm');
const formatString = minutes === 0 ? 'ha' : 'h:mma';
return time.format(formatString);
```

Reply:
Good tip, I think I can improve on that too.